Cargo.toml

```diff
@@ -38,9 +38,9 @@ ic-metrics-encoder = "1"
 ic-representation-independent-hash = "2.2"
 ic-response-verification = "2.2"
 ic-stable-structures = "0.6"
-ic-test-state-machine-client = "3"
 ic-crypto-standalone-sig-verifier = { git = "https://github.com/dfinity/ic", rev = "e69bcc7b319cbb3ebc22ec55af35287741244db6" }
 ic-types = { git = "https://github.com/dfinity/ic", rev = "e69bcc7b319cbb3ebc22ec55af35287741244db6" }
+pocket-ic = "4.0"
```

Comment:
@frederikrothenberger  in the tests it download `5.0.0` but here it uses `4`?

Reply:
Hi @cybrowl

According to the PocketIC compatibility matrix, it is correct to use the library version `4` with the server version `5`: https://docs.google.com/document/d/1VYmHUTjrgbzRHtsAyRrI5cj-gWGs7ktTnutPvUMJioU/edit?usp=sharing